certbot/certbot/compat/_path.py

```diff
@@ -29,7 +29,7 @@
 
 
 # Function os.path.realpath is broken on some versions of Python for Windows.
-def realpath(*unused_args, **unused_kwargs):
+def realpath(*unused_args, **unused_kwargs):  # type: ignore
```

Comment:
I personally think it'd be a good idea to include a comment explaining the use of `type: ignore` here.

Reply:
Done.